Cargo.toml

```diff
@@ -6,7 +6,7 @@ rust-version = "1.65"
 
 [features]
 default = ["tls-ring"]
-gperftools = ["dep:gperftools"]
+jemalloc = ["dep:tikv-jemallocator", "dep:jemalloc_pprof"]
```

Comment:
should we default to jemalloc? and/or unconditionally use it? or do we think we need more performance testing?

Reply:
For now I would rather not, this is just a drop-in replacement for what we had before that works, but we can certainly look into doing that later, yeah.